<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="mainTabWidget" name="mainTab" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openGraphAction"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Настройка</string>
    </property>
    <addaction name="actionNewObject"/>
    <addaction name="actionSerialPorts"/>
    <addaction name="actionNetSetting"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewObject">
   <property name="text">
    <string>Создать новый объект</string>
   </property>
  </action>
  <action name="openGraphAction">
   <property name="text">
    <string>Открыть график</string>
   </property>
  </action>
  <action name="actionSerialPorts">
   <property name="text">
    <string>Последовательные порты</string>
   </property>
  </action>
  <action name="actionNetSetting">
   <property name="text">
    <string>Настройка сети</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mainTabWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/maintabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
